public/view_lecture_tas.php

```diff
@@ -0,0 +1,91 @@
+<?php require_once(dirname(__FILE__)."/../modules/ensure_logged_in.php"); ?>
+
+<?php
+require_once "../modules/models/user.php";
+require_once "../modules/models/section.php";
+require_once "../modules/models/section_student.php";
+require_once "../common.php";
+
+$lecture_page_id = $_GET['id'];
+
+try {
+    $course_sections = Section::includes(["section_ta" => "user", "lecture" => []])->where(array('lecture_id' => $lecture_page_id));
+} catch (PDOException $error) {
+    echo "<br>" . $error->getMessage();
+}
+
+?> 
+<h2>Teaching Assistants</h2>
+<?php
+
+foreach ($course_sections as $course_section) {
+    echo "<h4>{$course_section->lecture->lecture_code} - {$course_section->section_code}</h3>";
```

Comment:
opening with h4 closing with h3 here

Reply:
nice catch not sure how you spotted that tho 😂